internal/agent/upgrade.go

```diff
@@ -64,7 +67,14 @@ func Upgrade(version, image string, force, debug bool) error {
 		discoveredImage = r.Data
 	})
 
-	_, err := bus.Manager.Publish(events.EventVersionImage, &events.VersionImagePayload{
+	options := map[string]string{}
```

Comment:
ditto, this is not correct - you have to scan the system to get the config, options are just returned by providers for additional args injection